### DISCUSSION TITLE
Option to interpolate volume between two sample control points

### DISCUSSION THREAD

**Discussion by MightyHelper** [ppy/osu#13091] [Ideas]
**Describe the new feature:** 
The name says it all:
Add a button that will do a linear interpolation of the volume between the volumes of 2 time frames.

This would make some volume transitions more natural.
**Proposal designs of the feature:** 
Add a new checkbox in the time frame options.

**Comment by smoogipoo** [comment]
What is a "time frame"? Do you mean global game volume or in-game (control point) volume?

**Comment by MightyHelper** [comment]
I meant the timing section time frames, where you can specify volume for hit objects, kiai, samples, etc.
What would be the right terminology?

**Comment by smoogipoo** [comment]
The correct terminology is sample control points. I've adjusted the PR title.
